extras/rya.geoindexing/src/test/java/org/apache/rya/indexing/geotemporal/mongo/MongoITBase.java

```diff
@@ -48,7 +48,7 @@ public void setupTest() throws Exception {
         conf.setMongoInstance("testDB");
         conf.setMongoPort("27017");
 
-        mongoClient = MockMongoFactory.newFactory().newMongoClient();
+        mongoClient = MockMongoSingleton.getInstance();
```

Comment:
Does GeoTemporalTestBase exist because MongoITBase isn't in an IT project?  This might be beyond the scope of this PR, but it might be good to create a ticket to put MongoITBase in a test project so that it can be used outside of its current project.